includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1279,13 +1281,19 @@ public function wc_setup_shipping_save() {
 
 			// Save chosen shipping method settings (using REST controller for convenience).
 			if ( ! empty( $_POST['shipping_zones']['domestic'][ $domestic_method ] ) ) { // WPCS: input var ok.
+
+				// Sanitize the cost field.
+				$domestic_cost = wc_clean( wp_unslash( $_POST['shipping_zones']['domestic'][ $domestic_method ] ) );
+				$domestic_cost = str_replace( array( get_woocommerce_currency_symbol(), html_entity_decode( get_woocommerce_currency_symbol() ) ), '', $domestic_cost );
```

Comment:
Since the primary reason for this PR is to change the sanitization on the flat rate shipping method, I don't think we should make a change to the wizard that would affect others. I agree that the sanitization here could be a lot better though!

Reply:
This will work fine.